File: Kabel Telepon/Sol.cpp
#include<bits/stdc++.h>
using namespace std;
typedef long long int ll;

int main(){
	int n;
	ll sum,x;
	cin>>n;
	for(int i=0; i<n; i++){
		cin>>x;
		if(x<2)
			cout<<"Kasus #"<<i+1<<": Tidak ada kabel!"<<endl;
		else {
			sum=(x*(x-1))/2;
			cout<<"Kasus #"<<i+1<<": "<<sum<<" kabel"<<endl;
		}
	}
	
	return 0;
}

File: Kabel Telepon/Sol.py
n = int(input())
for i in range (0,n):
    x = int(input())
    if (x<2):
        print("Kasus #", i+1, ": Tidak ada kabel!", sep='')
    else:
        ans = int((x*(x-1))/2)
        print("Kasus #", i+1, ": ", ans, " kabel", sep='')